src/main/java/com/spring/app/controllers/CountryController.java

```diff
@@ -1,40 +1,38 @@
 package com.spring.app.controllers;
 
-import com.spring.app.CountryLevel;
 import com.spring.app.domain.Country;
 import com.spring.app.repos.CountryRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 
 import java.util.Map;
 
+/**
+ * CountryController
+ *
+ * @author lyubov
+ */
 @Controller
 public class CountryController {
     @Autowired
     private CountryRepository countryRepository;
```

Comment:
Repository should be here, I suggest you to create a separate service layer where you perform data transformation and so on...)